### PR TITLE
[LLM Runtime] neural_speed_replace_graph

### PR DESCRIPTION
**Pull request by intellinjun**
## Type of Change

feature or bug fix or documentation or others
API changed or not
yes
## Description
- remove runtime_graph
- replace to neural_speed
detail description 
JIRA ticket: xxx

## Expected Behavior & Potential Risk

the expected behavior that triggered by this PR 

## How has this PR been tested?

how to reproduce the test (including hardware information)

## Dependency Change?

any library dependency introduced or removed

### CODE REVIEW

**Comment by intellinjun** [comment]
image

**Comment by zhenwei-intel** [comment]
neural speed should be added into requirement.txt

**Review by zhenwei-intel — APPROVED** [review]
LGTM

**Comment by intellinjun** [comment]
> neural speed should be added into requirement.txt

you are right, I had added it ,but now we can't install neural_speed with "pip install neural_speed".

File: requirements.txt [hunk 1]
```diff
@@ -7,3 +7,4 @@ setuptools_scm[toml]>=6.2
 torch==2.1.0+cpu
 accelerate
 optimum-intel
+neural_speed
```
Comment (airMeng) [inline requirements.txt:10]: Current we don't have pypi package published, how about install from source?

```suggestion
git+https://github.com/intel/neural-speed.git
```

cc @VincyZhang

File: setup.py [hunk 2]
```diff
@@ -245,8 +244,7 @@ def check_submodules():
     if not SKIP_RUNTIME:
         check_submodules()
         ext_modules.extend([
-            CMakeExtension("intel_extension_for_transformers.neural_engine_py", "intel_extension_for_transformers/llm/runtime/deprecated/"),
-            CMakeExtension("intel_extension_for_transformers.llm.runtime.graph.mpt_cpp", "intel_extension_for_transformers/llm/runtime/graph/"),
+            #CMakeExtension("intel_extension_for_transformers.neural_engine_py", "intel_extension_for_transformers/llm/runtime/deprecated/"),
```
Comment (zhewang1-intc) [inline setup.py:247]: pls reserve neural-engine because customers need it to run some models like Bert, SD...

File: setup.py [hunk 3]
```diff
@@ -122,7 +122,6 @@ def build_extension(self, ext: CMakeExtension) -> None:
             f"-DDNNL_CPU_RUNTIME=OMP",
             f"-DNE_WITH_AVX2={'ON' if NE_WITH_AVX2 else 'OFF'}",
```
Comment (zhewang1-intc) [inline setup.py:123]: some cmake flags don't make sense once once graph remove from ITREX.

**Comment by zhewang1-intc** [comment]
pls note:
1. is there any other component of ITREX(e.g. neural-chat relies on llmruntime's python API, if so, pls update)
2. https://github.com/intel/intel-extension-for-transformers/pull/1126 will be merged very soon, pls delete library-dir after it merge(library dir contains 3rd-lib, pls delete it as well).

File: setup.py (hunk 2 above)
Comment (airMeng) [reply setup.py:247]: cc @a32543254

Comment (a32543254) [reply setup.py:247]: Yes, we will keep deprecated in ITREX

File: setup.py (hunk 3 above)
Comment (a32543254) [reply setup.py:123]: Yes, some cmake flag relate to cpp grape will be remove and relate to deprecate will be keep @intellinjun

Comment (intellinjun) [reply setup.py]: done

Comment (DDEle) [reply setup.py]: Done? I think the deprecate executor will use `DNNL_CPU_RUNTIME` and `NE_WITH_AVX2`.

Comment (intellinjun) [reply setup.py]: You are right, deprecate executor will use NE_WITH_AVX2, no DNNL_CPU_RUNTIME

**Review by airMeng — APPROVED** [review]

**Comment by airMeng** [comment]
please update https://github.com/intel/intel-extension-for-transformers/blob/main/.github/CODEOWNERS as well

**Comment by VincyZhang** [comment]
@intellinjun please resolve conflicts and CI will be triggered automatically

**Review by airMeng — CHANGES_REQUESTED** [review]
https://github.com/intel/intel-extension-for-transformers/tree/main/intel_extension_for_transformers/llm/library/jblas
why not remove jblas as well?

**Comment by intellinjun** [comment]
> https://github.com/intel/intel-extension-for-transformers/tree/main/intel_extension_for_transformers/llm/library/jblas why not remove jblas as well?

I will remove it soon,

**Review by zhewang1-intc — APPROVED** [review]
LGTM

File: .github/CODEOWNERS [hunk 4]
```diff
@@ -28,9 +28,6 @@
 /intel_extension_for_transformers/llm/runtime/deprecated/Cmake* yu.luo@intel.com
 /intel_extension_for_transformers/llm/runtime/deprecated/cmake/ yu.luo@intel.com
 /intel_extension_for_transformers/llm/library/jblas yu.luo@intel.com
-/intel_extension_for_transformers/llm/runtime/graph hengyu.meng@intel.com
-/intel_extension_for_transformers/llm/runtime/graph/scripts zhenwei.liu@intel.com
-/intel_extension_for_transformers/llm/runtime/graph/__init__.py zhenwei.liu@intel.com
 /intel_extension_for_transformers/llm/library hengyu.meng@intel.com
```
Comment (zhewang1-intc) [inline .github/CODEOWNERS:31]: as above

File: .github/CODEOWNERS [hunk 5]
```diff
@@ -28,9 +28,6 @@
 /intel_extension_for_transformers/llm/runtime/deprecated/Cmake* yu.luo@intel.com
 /intel_extension_for_transformers/llm/runtime/deprecated/cmake/ yu.luo@intel.com
 /intel_extension_for_transformers/llm/library/jblas yu.luo@intel.com
```
Comment (zhewang1-intc) [inline .github/CODEOWNERS:30]: remove it

**Review by airMeng — APPROVED** [review]

**Review by VincyZhang — APPROVED** [review]

**Pull request merged**